matrix.py

```diff
@@ -0,0 +1,47 @@
+from typing import Self
+
+
+class Matrix:
+    """
+    Matrix class.
+
+    Represents the mathematical matrix and implements the method of the sum of two matrix.
+
+    """
+
+    def __init__(self: Self, init_data: list[list[int | float]] | tuple[list[int | float], ...]):
+        """
+        Init method.
+
+        Takes the matrix parameters (width and height of a matrix) as a list of lists.
+        """
```

Comment:
```suggestion
        """Init method. """
```
It's better to write a simply docksting if there aren't need a special explanations to a hard buisness logic

Reply:
Fix in all code

Reply:
I fixed. See the latest changes